### PR TITLE
Update Tarteaucitron to v1.8.4

### PR DESCRIPTION
**Pull request by GaetanBt**

### CODE REVIEW

File: src/assets/script.js [hunk 1]
```diff
@@ -5,22 +5,29 @@
...
-      button.classList.add('btn', 'btn-primary', 'ml-2')
-    })
-
-    servicesContainer.querySelectorAll('.tarteaucitronDeny').forEach(function (button) {
-      button.classList.add('btn', 'btn-secondary', 'ml-2')
+    servicesContainer.querySelectorAll('button').forEach(function (button) {
+      if (button.classList.contains('tarteaucitronAllow')) {
+        button.classList.add('btn', 'btn-primary', 'btn-inverse', 'ml-2')
+      } else {
+        button.classList.add('btn', 'btn-secondary', 'btn-info', 'ml-2')
```
Comment (ffoodd) [inline src/assets/script.js:29]: Setting both `.btn-secondary` and `.btn-info` is redundant here: did you mean `.btn-inverse` maybe?

File: src/assets/script.js [hunk 2]
```diff
@@ -5,22 +5,29 @@
...
+
+    document.querySelectorAll('#tarteaucitronRoot button').forEach(function (button) {
+      if (button.classList.contains('catToggleBtn') || button.id === 'tarteaucitronClosePanel') {
+        return
+      }
+
+      if (button.classList.contains('tarteaucitronAllow')) {
+        button.classList.add('btn', 'btn-primary', 'btn-inverse', 'ml-2')
+      } else {
+        button.classList.add('btn', 'btn-inverse', 'btn-info', 'ml-2')
```
Comment (ffoodd) [inline src/assets/script.js:17]: Setting `.btn-inverse` as the second name would improve consistency thus legibility ;)

File: src/assets/script.js (hunk 1 above)
Comment (GaetanBt) [reply src/assets/script.js:29]: In fact, all I needed was `.btn-info` here, thanks!

File: src/assets/script.js (hunk 2 above)
Comment (GaetanBt) [reply src/assets/script.js]: Done

**Review by ffoodd — APPROVED** [review]
:shrug:

**Pull request merged**